### ISSUE TITLE
Cannot find bundler

### ISSUE THREAD

**Issue by dan-wittenberg** [puppetlabs/pdk#166]
Testing 0.4.2 running 'pdk validate' results in:
```
$ pdk validate -d    
pdk (INFO): Running all available validators...
pdk (DEBUG): PDK package installation not found, trying 'bundle' from the system PATH instead
[-] Checking for missing Gemfile dependenciespdk (DEBUG): Executing 'bundle check --gemfile=/home/dwittenberg/github/modules/ahead/Gemfile --path=/home/dwittenberg/.pdk/cache'
[���] Checking for missing Gemfile dependencies

/opt/puppetlabs/puppet/lib/ruby/2.1.0/rubygems/dependency.rb:298:in `to_specs': Could not find 'bundler' (>= 0) among 8 total gem(s) (Gem::LoadError)
	from /opt/puppetlabs/puppet/lib/ruby/2.1.0/rubygems/dependency.rb:309:in `to_spec'
	from /opt/puppetlabs/puppet/lib/ruby/2.1.0/rubygems/core_ext/kernel_gem.rb:53:in `gem'
	from /opt/puppetlabs/puppet/bin/bundle:22:in `<main>'
pdk (DEBUG): PDK package installation not found, trying 'bundle' from the system PATH instead
[-] Installing missing Gemfile dependenciespdk (DEBUG): Executing 'bundle install --gemfile=/home/dwittenberg/github/modules/ahead/Gemfile --path=/home/dwittenberg/.pdk/cache'
[���] Installing missing Gemfile dependencies

/opt/puppetlabs/puppet/lib/ruby/2.1.0/rubygems/dependency.rb:298:in `to_specs': Could not find 'bundler' (>= 0) among 8 total gem(s) (Gem::LoadError)
	from /opt/puppetlabs/puppet/lib/ruby/2.1.0/rubygems/dependency.rb:309:in `to_spec'
	from /opt/puppetlabs/puppet/lib/ruby/2.1.0/rubygems/core_ext/kernel_gem.rb:53:in `gem'
	from /opt/puppetlabs/puppet/bin/bundle:22:in `<main>'
pdk (FATAL): Unable to install missing Gemfile dependencies.
pdk (DEBUG): /opt/puppetlabs/puppet/lib/ruby/gems/2.1.0/gems/pdk-0.4.2/lib/pdk/util/bundler.rb:32:in `ensure_bundle!'
pdk (DEBUG): /opt/puppetlabs/puppet/lib/ruby/gems/2.1.0/gems/pdk-0.4.2/lib/pdk/cli/validate.rb:77:in `block (2 levels) in <module:CLI>'
pdk (DEBUG): /opt/puppetlabs/puppet/lib/ruby/gems/2.1.0/gems/cri-2.9.1/lib/cri/command.rb:319:in `call'
pdk (DEBUG): /opt/puppetlabs/puppet/lib/ruby/gems/2.1.0/gems/cri-2.9.1/lib/cri/command.rb:319:in `run_this'
pdk (DEBUG): /opt/puppetlabs/puppet/lib/ruby/gems/2.1.0/gems/cri-2.9.1/lib/cri/command.rb:264:in `run'
pdk (DEBUG): /opt/puppetlabs/puppet/lib/ruby/gems/2.1.0/gems/cri-2.9.1/lib/cri/command.rb:282:in `run'
pdk (DEBUG): /opt/puppetlabs/puppet/lib/ruby/gems/2.1.0/gems/pdk-0.4.2/lib/pdk/cli.rb:15:in `run'
pdk (DEBUG): /opt/puppetlabs/puppet/lib/ruby/gems/2.1.0/gems/pdk-0.4.2/exe/pdk:5:in `<top (required)>'
pdk (DEBUG): /opt/puppetlabs/puppet/bin/pdk:23:in `load'
pdk (DEBUG): /opt/puppetlabs/puppet/bin/pdk:23:in `<main>'

$ which gem
/opt/puppetlabs/puppet/bin/gem

$ gem list |grep -i bundler
bundler (1.15.2, 1.15.1, 1.13.7)
```

**Comment by scotje** [comment]
Should be resolved by #170

**Comment by scotje** [comment]
Will be making a new gem release shortly.
